movies/models.py

```diff
@@ -11,7 +11,7 @@
 class Movie(models.Model):
     user = models.ForeignKey(User, null=True, on_delete=models.CASCADE)
     name = models.CharField(max_length=100, null=True, verbose_name='Title')
-    poster = models.ImageField(verbose_name='Poster')
+    poster = models.URLField(verbose_name='Poster')
```

Comment:
Why not keep it as an ImageField?
https://stackoverflow.com/questions/64269431/django-best-practice-for-storing-images-urlfield-vs-imagefield